Lib/test/test_asyncio/test_events.py

```diff
@@ -38,6 +38,12 @@
     from asyncio import test_support as support
 
 
+if ssl is not None:
+    IS_OPENSSL_1_1 = ssl.OPENSSL_VERSION_INFO >= (1, 1, 0)
```

Comment:
Woudl a oneliner be more readable? Or less? Something like:

```python
IS_OPENSSL_1_1 = ssl is not None and ssl.OPENSSL_VERSION_INFO >= (1, 1, 0)
```

Or even:


```python
IS_OPENSSL_1_1 = ssl and ssl.OPENSSL_VERSION_INFO >= (1, 1, 0)
```

Reply:
I prefer to write the code on 4 lines ;-)